build/commands/lib/util.js

```diff
@@ -555,6 +555,11 @@ const util = {
     ]
 
     if (config.use_goma) {
+      // Setting `AUTONINJA_BUILD_ID` allows tracing Goma remote execution which helps with
+      // debugging issues (e.g., slowness or remote-failures).
+      options.env.AUTONINJA_BUILD_ID = config.gomaBuildId
```

Comment:
this is not goma specific so we just just always set it for autoninja

Reply:
done

Reply:
sorry @Yannic, I didn't mean to move it back to buildTarget, I mean that it shouldn't be inside ` if (config.use_goma)`